test/bigtest/tests/units/settings/MappingProfileDetails/MappingProfileDetails-test.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+import { noop } from 'lodash';
+import sinon from 'sinon';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Paneset } from '@folio/stripes/components';
+import { mountWithContext } from '@folio/stripes-data-transfer-components/interactors';
+import commonTranslations from '@folio/stripes-data-transfer-components/translations/stripes-data-transfer-components/en';
+import stripesComponentsTranslations from '@folio/stripes-components/translations/stripes-components/en';
+
+import translations from '../../../../../../translations/ui-data-export/en';
+import { MappingProfileDetails } from '../../../../../../src/settings/MappingProfiles/MappingProfileDetails';
+import { MappingProfileDetailsInteractor } from './interactors/MappingProfileDetailsInteractor';
+
+describe('MappingProfileDetails', () => {
+  const translationsProperties = [
+    {
+      prefix: 'ui-data-export',
+      translations,
+    },
+    {
+      prefix: 'stripes-components',
+      translations: stripesComponentsTranslations,
+    },
+  ];
+  const mappingProfileDetails = new MappingProfileDetailsInteractor();
+
+  describe('rendering mapping profile details', function () {
+    const handleSubmitSpy = sinon.spy();
+
+    beforeEach(async function () {
+      handleSubmitSpy.resetHistory();
+
+      await mountWithContext(
+        <Paneset>
+          <Router>
+            <MappingProfileDetails
+              mappingProfile={{
+                hasLoaded: true,
+                records: [{
+                  id: '25d81cbe-9686-11ea-bb37-0242ac130002',
```

Comment:
I think, it was taken from 'fetch-mapping-profiles-success.js" file and updated by transformations. I wonder if we can create 1 object and use it in the both cases. It seems, it can make easier to maintain it in the future.

Reply:
Good idea! Update it